.gitlab-ci.yml

```diff
@@ -58,12 +58,23 @@ before_script:
   CLOUD_MACHINE_TYPE: "g1-small"
   CI_PLATFORM: "gce"
   PRIVATE_KEY: $GCE_PRIVATE_KEY
+  SSH_KEY_PATH: gce.private
+
+.packet_variables: &packet_variables
+  SSH_USER: root
+  CLOUD_MACHINE_TYPE: "g1-small"
```

Comment:
There is no g1-small instance on packet. I dont think this var is used, else it would be `t1-small`